.travis.yml

```diff
@@ -14,8 +14,7 @@ jobs:
       - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
       - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.21.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
       # Install nsenter
-      - docker build -t ubuntu-nsenter ./hack/ubuntu-nsenter
-      - docker run -v /usr/local/bin:/hostbin ubuntu-nsenter cp /nsenter /hostbin/nsenter
+      - docker run -v /usr/local/bin:/hostbin munnerz/ubuntu-nsenter cp /nsenter /hostbin/nsenter
```

Comment:
* This should be a Jetstack image.
* Maybe make it a Dockerhub automated build? https://docs.docker.com/docker-hub/builds/
* Or at least add a note to the dockerhub page explaining where the Dockerfile is.